packages/components/InputPanel/InputPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex, Grid } from '@chakra-ui/react';
+
+import { pass } from '@packages/utils/pass';
+
+import BaseButton from '../BaseButton';
+
+import { NumericButton } from './NumericButton';
+
+const generateNumberList = (max = 10) => {
+  return Array(max)
+    .fill(null)
+    .map((_, i) => (i + 1) % max);
+};
+
+export default function InputPanel() {
+  return (
+    <Flex
+      h="100%"
+      direction="column"
+      alignItems="center"
+      justifyContent="center"
+      px="2"
+      backgroundColor="blackAlpha.700"
+    >
+      <Grid
+        templateColumns="repeat(3, 1fr)"
+        h="30%"
+        w="50%"
+        gap="1"
+        justifyItems="center"
+        alignContent="center"
+      >
+        {generateNumberList().map((num) => (
+          <NumericButton key={num} onclick={pass}>
+            {String(num)}
```

Comment:
what is a String in this context? Is it a type or is variable name?

Reply:
it is a function that takes an argument and parses it into a string.
Ps. It is a built-in function

Reply:
Acho que poderia ser um inteiro direto